syntax = "proto3";


message Owner {
    // The agent's unique public key
    string public_key = 1;

    // A human-readable name identifying the reader
    string name = 2;

    // Approximately when the reader was registered, as a Unix UTC timestamp
    uint64 timestamp = 3;
}


message OwnerContainer {
    repeated Owner entries = 1;
}
